### PR TITLE
Memo menu

### PR DESCRIPTION
**Pull request by RabbitDoge**
Reduce the number of re render on the menu

### CODE REVIEW

**Review by hachiojidev — APPROVED** [review]

File: src/widgets/Menu/components/Accordion.tsx [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-import { ArrowDropDownIcon, ArrowDropUpIcon } from "../../components/Svg";
+import { PushedProps } from "../types";
+import { ArrowDropDownIcon, ArrowDropUpIcon } from "../../../components/Svg";
 
 interface Props extends PushedProps {
   label: string;
   icon: React.ReactElement;
   initialOpenState?: boolean;
   className?: string;
+  children: unknown;
```
Comment (hachiojidev) [inline src/widgets/Menu/components/Accordion.tsx:13]: Reason for this over `ReactNode`?

File: src/widgets/Menu/icons/Logo.tsx [hunk 2]
```diff
@@ -82,4 +82,4 @@ const Logo: React.FC<LogoProps> = ({ isDark, ...props }) => {
   );
 };
 
-export default Logo;
+export default React.memo(Logo, (prev, next) => prev.isDark === next.isDark);
```
Comment (hachiojidev) [inline src/widgets/Menu/icons/Logo.tsx:85]: Do you think it is necessary?

File: src/widgets/Menu/components/Accordion.tsx (hunk 1 above)
Comment (RabbitDoge) [reply src/widgets/Menu/components/Accordion.tsx:13]: ReactNode sounds better

File: src/widgets/Menu/icons/Logo.tsx (hunk 2 above)
Comment (RabbitDoge) [reply src/widgets/Menu/icons/Logo.tsx:85]: if the parent re render but the isDark value doesn't change, this component will rerender

**Pull request merged**